src/commands/admin.rs

```diff
@@ -235,3 +254,91 @@ pub async fn member_count(ctx: &Context, msg: &Message, mut args: Args) -> Comma
         .await?;
     Ok(())
 }
+
+#[command]
+#[description("Mute a user for 12h or the specified time in hours")]
+#[usage("@user [time]")]
+#[min_args(1)]
+pub async fn mute(ctx: &Context, msg: &Message, mut args: Args) -> CommandResult {
```

Comment:
It would be nice if the command allowed us to mute a user for less than an hour.
In case of meaningless spam one hour mute seems a bit overkill